rbi/core/array.rbi

```diff
@@ -150,7 +150,7 @@ class Array < Object
     params(
         arg0: Integer,
     )
-    .returns(T::Enumerator[Elem])
+    .returns(T::Enumerator[T::Array[Elem]])
```

Comment:
@carl-stripe can you update both halves of the overload? or is there some reason why the first overload is still correct?

Reply:
This line: https://github.com/sorbet/sorbet/pull/1131/files#diff-41b3206380d3b7469fde9a3fc6c8129eR147

Reply:
jk i see that that half of the overload is actually correct.

Reply:
```
[1] pry(main)> [1,2,3].combination(2){|x| }
=> [1, 2, 3]
```